Slack.Automation/Promact.Core.Repository/ExternalLoginRepository/OAuthLoginRepository.cs

```diff
@@ -116,8 +116,28 @@ public OAuthApplication ExternalLoginInformation(string refreshToken)
         /// <param name="slackEvent"></param>
         public void SlackEventUpdate(SlackEventApiAC slackEvent)
         {
-            slackEvent.Event.User.TeamId = slackEvent.TeamId;
-            _slackUserDetails.Insert(slackEvent.Event.User);
+            var user = _slackUserDetails.FirstOrDefault(x => x.UserId == slackEvent.Event.User.UserId);
+            if (user == null)
```

Comment:
What happens if a user updated any of the property? like if he updated his 1st name or last name. Will you not update user at your end?